core/app.go

```diff
@@ -343,6 +349,9 @@ func (a *App) handlePolling() {
 		}
 		a.Telemetry.Serve()
 	}
+	if a.ControlSocket != nil {
```

Comment:
> Would you also like a CLI flag to override the default FS location of the socket file?

We've put all the config in the config file rather than flags (except for the optional config file location). I think for consistency we should keep it that way.